### DISCUSSION TITLE
How to use OfficeDevPnP.Core with new PnP.Powershell?

### DISCUSSION THREAD

**Discussion by evlo** [pnp/powershell#321] [Q&A] [answered]
We used OfficeDevPnP.Core (well mostly OfficeDevPnP.Core.Enums really, but anyway) with old SharePointPnPPowerShellOnline, now migrating to PnP.Powershell - what is best practice to do it?
Will this https://github.com/pnp/pnpcore be the new thing? And if so what is recommended approach to use OfficeDevPnP.Core with new PnP.Powershell until https://github.com/pnp/pnpcore is in V1?

Update: I guess this https://github.com/pnp/pnpframework is the correct new thing, right?

Only way to get the enums so far is `Add-Type -Path "C:\Program Files\WindowsPowerShell\Modules\PnP.PowerShell\1.3.0\Framework\PnP.Framework.dll"`

**Comment by gautamdsheth** [comment, marked as answer]
@evlo - Yes, that's correct,  PnP.Framework.dll should be used for your enums. PnP PowerShell depends on this for some commandlets.

Can you elaborate on why you need these enums ? They should be available I guess without explicitly loading the module.

**Comment by evlo** [comment]
Ie. `PnP.Framework.Enums.AppCatalogScope`
